components/api_calls/rpms_images_fetcher_calls.js

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+
+let server_endpoint = null
+
+if (process.env.NEXT_PUBLIC_RUN_ENV === "dev") {
+    server_endpoint = "http://localhost:8080/"
+} else {
+    server_endpoint = process.env.NEXT_PUBLIC_ART_DASH_SERVER_ROUTE + "/"
+
+    // OPENSHIFT_BUILD_NAMESPACE env variable is obtained inside pod during its run
+    server_endpoint = server_endpoint.replace(/\{0\}/g, process.env.NEXT_PUBLIC_OPENSHIFT_BUILD_NAMESPACE);
```

Comment:
Why do we need the `OPENSHIFT_BUILD_NAMESPACE` variable?

Reply:
good question! :D